plugins/httpapi/restconf.py

```diff
@@ -73,9 +73,10 @@ def send_request(self, data, **message_kwargs):
 
 def handle_response(response, response_data):
     try:
-        response_data = json.loads(response_data.read())
+        res = response_data.read().strip()
```

Comment:
```suggestion
    res = response_data.read().strip()
    try:
```
(It won't let me remove the `try` above, but you get the idea)

The reason it's split into response and response_data is that the latter is a buffer containing the read-once data from the former, so you _should_ be able to `response_data.seek(0)` and get the same result. But this is just fine, too.